language/reserved_keywords.rb

```diff
@@ -0,0 +1,155 @@
+require_relative '../spec_helper'
+
+describe "Ruby's reserved keywords" do
+  # Copied from Prism::Translation::Ripper
+  keywords = [
```

Comment:
TODO: It would be nice to have one canonical list that can be used in all these places:

1. `spec/ruby/language/hash_spec.rb`
2. `spec/ruby/language/reserved_keywords.rb`
3. `lib/prism/translation/ripper.rb`
4. Ideally, it would also be nice to generate `doc/keywords.rdoc` from that list, to ensure that all keywords are documented.

Where should such a canonical list live?

Reply:
There is no way to avoid duplication there, since it's files in different repositories.
So for ruby/spec let's have it as a new file under `language/fixtures`.
I think a `%w[]` array would be nice for this BTW.

FWIW in ruby/ruby there is also `defs/keywords`.